api_tests/lib/actor.ts

```diff
@@ -80,7 +80,7 @@ export class Actor {
         }
     }
 
-    public doComitAction(action: Action): Promise<Response> {
+    public async doComitAction(action: Action): Promise<Response> {
```

Comment:
Oh my god this is so nice, it detected that a `Promise` was returned and hence told you to add `async`?

Reply:
Yes :)